tests/e2e/actor_features/actor_features_test.go

```diff
@@ -286,4 +287,23 @@ func TestServiceInvocation(t *testing.T) {
 
 		require.True(t, anyActorMoved)
 	})
+
+	t.Run("Get actor metadata", func(t *testing.T) {
+		// Each test needs to have a different actorID
```

Comment:
Let's scale app to 1 first because the previous tests scale out to 2 instances.

```
tr.Platform.Scale(appName, 1)
```

If it works, it means that the table was not updated from placement service when getting actors count.